iceoryx_posh/test/moduletests/test_roudi_iceoyx_roudi_memory_manager.cpp

```diff
@@ -76,5 +76,47 @@ TEST_F(IceoryxRoudiMemoryManager_test, createAndAnnouceMemorySuccess)
     EXPECT_THAT(testResult.has_error(), Eq(false));
 }
 
+/// @brief test to check function MgmtMemoryProviderSuccess
+TEST_F(IceoryxRoudiMemoryManager_test, MgmtMemoryProviderSuccess)
```

Comment:
Our test case names are following the following scheme: WhatDoYouTestWithSpecifiedOutcome
So the test here would be named something like:
`CreatingMgmtProviderSuccessfullyReturnsNonNullptr`

Could you please rename this and all the other tests accordingly.